src/libraries/System.Net.Http/tests/FunctionalTests/SocketsHttpHandlerTest.cs

```diff
@@ -1634,6 +1634,8 @@ public sealed class SocketsHttpHandler_HttpClientHandler_MaxResponseHeadersLengt
     }
 
     [ConditionalClass(typeof(HttpClientHandlerTestBase), nameof(IsQuicSupported))]
+    [ActiveIssue("https://github.com/dotnet/runtime/issues/94507")]
```

Comment:
```suggestion
```

If it's the same root cause, should we consolidate to one issue?
E.g. close #94507 + edit the pattern in #101015 to match all methods on the class

Reply:
We can, but I don't have any evidence that it's the same root cause, so I'd like to keep both of them open for now.